python/lsst/pipe/tasks/drpAssociationPipe.py

```diff
@@ -275,7 +275,10 @@ def visitDetectorPair(dataRef):
                 unassociatedSsObjects = unassociatedSsObjects[ssObjInTractPatch]
                 nSsSrc = ssInTractPatch.sum()
                 nSsObj = ssObjInTractPatch.sum()
-                diaCat = ssoAssocResult.unAssocDiaSources.to_pandas()
+                if len(ssoAssocResult.unAssocDiaSources) > 0:
```

Comment:
This is fine to leave and perfectly readable, but in the future, we prefer to use the fact that empty sequences are `False`: https://developer.lsst.io/python/style.html#is-and-is-not-should-be-used-when-comparing-to-none